libkbfs/tlf_journal.go

```diff
@@ -813,6 +826,56 @@ func (e errTLFJournalNotEmpty) Error() string {
 	return "tlfJournal is not empty"
 }
 
+func (j *tlfJournal) checkServerForConflicts(ctx context.Context) error {
```

Comment:
We don't have a convention for it in this file, and there are many other functions with the same issue.  I'm inclined to favor consistency with those other functions for now.